tests/run/numpy_test.pyx

```diff
@@ -944,3 +944,15 @@ def test_broadcast_comparison(np.ndarray[double, ndim=1] a):
     cdef object obj = a
     return a == 0, obj == 0, a == 1, obj == 1
 
+
+@testcase
+def test_c_api_searchsorted():
+    cdef:
+        np.ndarray arr = np.random.randn(10)
+
+    other = np.random.randn(5)
+
+    result = np.PyArray_SearchSorted(arr, other, np.NPY_SEARCHRIGHT, NULL)
+
+    expected = arr.searchsorted(other, side="right")
+    import (result == expected).all()
```

Comment:
`assert`, maybe?
Also, you need to write a doctest here to run this test.
And, if this test fails, the test output will most likely not be helpful, so maybe return the comparison result as a list instead and let the doctest print it?

Reply:
> assert, maybe?

yikes, yah.  will update.